src/_includes/components/calls-to-action/calls-to-action.njk

```diff
@@ -0,0 +1,7 @@
+<div class="calls-to-action calls-to-action--{{ params.variantClass }}">
```

Comment:
The class should be `call-to-action` for a singular item. So this will need to be changed throughout.